app/views/shared/search/_radius.html.slim

```diff
@@ -3,7 +3,7 @@
   :last,
   :first,
   label: { text: t("jobs.search.within_radius"), size: "s" },
-  class: "govuk-!-width-#{wide ? 'full' : 'one-third'}",
+  class: "govuk-!-width-#{wide ? 'full' : 'one-half'}",
```

Comment:
It looks like you've removed the need for us to have a `wide` variable for two out of three search input partials (the other two: keyword and location). So it would be nice to stop setting the variable where it's not needed:

https://github.com/DFE-Digital/teaching-vacancies/blob/be355535078eff4f3254e925beb63efc59d209b9/app/views/home/_search.html.slim#L6
https://github.com/DFE-Digital/teaching-vacancies/blob/be355535078eff4f3254e925beb63efc59d209b9/app/views/home/_search.html.slim#L7
https://github.com/DFE-Digital/teaching-vacancies/blob/be355535078eff4f3254e925beb63efc59d209b9/app/views/vacancies/_search.html.slim#L50
https://github.com/DFE-Digital/teaching-vacancies/blob/be355535078eff4f3254e925beb63efc59d209b9/app/views/vacancies/_search.html.slim#L51

Reply:
good point yeah can remove these now 👍